Makefile

```diff
@@ -90,6 +90,9 @@ minikube-start:
 minikube-run-local: build
 	build/minikube/run-local.sh ${NAMESPACE}
 
+minikube-test: build
```

Comment:
TIP: If add a comment on top of the minikube related tasks, when someone types ```make``` it shows a summary of available targets.